docs/node/run-your-node/paratime-node.mdx

```diff
@@ -114,75 +93,53 @@ change in the future.
 [Oasis CLI tools]: ../../general/manage-tokens/cli/account.md#delegate
-[Oasis Node Stake Account Info]: ../../general/manage-tokens/cli/account.md#show
+[`oasis account show`]: ../../general/manage-tokens/cli/account.md#show
 
 ### Register a New Entity or Update Your Entity Registration
 
-If you don't have an entity yet, create a new one by following the [Creating Your Entity](validator-node.mdx#creating-your-entity) instructions.
-
 :::danger
 
-Everything in this section should be done on the `localhost` as there are sensitive items that will be created.
+Everything in this section should be done on the `localhost` as there are
+sensitive items that will be created.
 
 :::
 
-If you will be running the ParaTime on a new Oasis Node, initialize a new node by following the [Initializing a Node](validator-node.mdx#initializing-a-node) instructions.
+1. If you don't have an entity yet, create a new one by following the
+   [Initialize Entity] instructions for validators.
 
-Then update your entity descriptor by enumerating paths to all your node's descriptors (existing and new ones) in the `--entity.node.descriptor` flag. For example:
-
-```bash
-oasis-node registry entity update \
-    ... various signer flags ... \
-    --entity.node.descriptor /localhost/node1/node_genesis.json,/localhost/node2/node_genesis.json
-```
-
-:::info
-
-To confirm all nodes are added to your entity descriptor, run:
-
-```bash
-cat <PATH-TO-entity.json>
-```
+2. If you will be running the ParaTime on a new Oasis node, also initialize a
+   new node by following the [Starting the Oasis Node] instructions for
+   validators.
 
-and ensure you see all your nodes' IDs under the `"nodes"` list.
+3. Now, [list your node ID] in the entity descriptor file `nodes` field.
 
-For example:
-
-```bash
-{
-  "v": 2,
-  "id": "QTg+ZjubD/36egwByAIGC6lMVBKgqo7xnZPgHVoIKzc=",
-  "nodes": [
-    "yT1h8/eN0VInQxn3YKcHxvSgGcsjsTSYmdTLZZMBTWI=",
-    "HG5TB3dbY8gtYBBw/R/cHfPaOpe0vT7W1wu/2rtpk/A="
-  ]
-}
-```
-
-:::
-
-Then generate and submit the new/updated entity descriptor via the entity registration transaction by following the [Generating Entity Registration Transaction](validator-node.mdx#generating-entity-registration-transaction) instructions.
-
-:::caution
-
-Make sure your entity descriptor (i.e. `entity.json`) is copied to your online server and saved as `/node/entity/entity.json` to ensure the [node's configuration](paratime-node.mdx#configuration) will find it.
-
-:::
+4. [Register] the updated entity descriptor.
 
 :::tip
 
-You will [configure the node](paratime-node.mdx#configuration) to automatically register for the roles it has enabled (i.e. storage and compute roles) via the `worker.registration.entity` configuration flag.
+You will [configure the node](#configuration) to automatically
+register for the roles it has enabled (i.e. storage and compute roles) via the
+`worker.registration.entity` configuration flag.
```

Comment:
```suggestion
`registration.entity` configuration flag.
```
With v23 the configuration was moved.

Maybe change to an example config snippet for the lazy?

Reply:
I moved the configuration change to a separate PR https://github.com/oasisprotocol/docs/pull/615.